virtinst/devices/disk.py

```diff
@@ -979,6 +979,8 @@ def _default_bus(self, guest):
             return "sd"
         if guest.os.is_q35():
             return "sata"
+        if self.is_cdrom() and guest.os.is_loongarch64():
+            return "scsi"
```

Comment:
this line isn't hit by the test case. have one of your test_cli.py cases add a `--disk device=cdrom` and `pytest --regenerate-output` to update the test content

Reply:
Sorry, I did not do the above operation in test_cli, but I think it is unnecessary to add these statements, Because it has the following statement on it：

`
if (self.is_cdrom() and
     guest.supports_virtioscsi() and
     not guest.os.is_x86()):
     return "scsi"
`
So I will delete the previously added statement when I push patch next time.